conf/clean_up_repo_conf.py

```diff
@@ -0,0 +1,44 @@
+"""
+The file will have relative paths for dir and respective files which clean_up_repo.py will delete.
+"""
+import os
+
+# Declaring directories as directory list
+# dir_list : list
+
```

Comment:
Do we need this whitespace/empty line?

Reply:
@qxf2 , This is already being followed in our framework.  e.g.`copy_framework_template.py` 

```
"""
This script would copy the required framework files from the input source to the input destination given by the user.
1. Copy files from POM to the newly created destination directory.
2. Verify if the destination directory is created and create the sub-folder to copy files from POM\Conf.
3. Verify if the destination directory is created and create the sub-folder to copy files from POM\Page_Objects.
4. Verify if the destination directory is created and create the sub-folder to copy files from POM\Utils.

"""
import os,sys
import shutil
from optparse import OptionParser
sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
from conf import copy_framework_template_conf as conf

def copy_framework_template(src_folder,dst_folder):
	"run the test"
	#1. Copy files from POM to the newly created destination directory.
	#1a. Get details from conf file
	src_files_list = conf.src_files_list
		
	#1b. Create the new destination directory
	os.makedirs(dst_folder)
```
Hence I have to follow it in this file as well.

Reply:
@qxf2 , let me know in case you have more comments around this.